files/id-stg/docker-compose.yml

```diff
@@ -1,40 +1,44 @@
-version: "3"
+version: '3'
+
 services:
-  mongodb:
-    restart: "always"
-    image: mongo:3.4.9
+  db:
+    image: postgres
+    environment:
+      POSTGRES_DB: ${POSTGRES_DB}
+      POSTGRES_USER: ${POSTGRES_USER}
+      POSTGRES_PASSWORD: ${POSTGRES_PASSWORD}
     healthcheck:
-      test: ["CMD", "mongo", "--quiet", "localhost/test", "--eval", "'quit(db.runCommand({ ping: 1 }).ok ? 0 : 1)';"]
-      interval: 1m30s
-      timeout: 10s
-      retries: 3
-    volumes:
-      - mongo-data:/data/db
-      - mongo-backup:/opt/backup
-
-  id-web:
```

Comment:
I'm a little nervous that this is removing the ID app altogether and, while that's the eventual goal, I'm not sure if we're there yet (specifically, do we support Discourse at all?).

Reply:
Hello @ibacher,
This PR makes updates to the staging environment, which will be deployed at http://id-stg.openmrs.org/.

Reply:
@ibacher thanks for bringing this up! I thought we are getting rid of the id app and using Keycloak as a replacement. Discourse has an SSO plugin that works perfectly with Keycloak. Anything else that relies on the id app?

Reply:
I think that’s it, aside from documentation. And, yeah, I’m all in favour of killing the ID app; just want to make sure we’re doing it intentionally.